src/RokuAdapter.ts

```diff
@@ -156,9 +153,10 @@ export class RokuAdapter {
     /**
      * Connect to the telnet session. This should be called before the channel is launched.
      */
-    public async connect(enableDebuggerAutoRecovery: boolean = false) {
+    public async connect(enableDebuggerAutoRecovery: boolean = false, enableLookupVariableNodeChildren: boolean = false) {
```

Comment:
@georgejecook Is there a reason you add these parameters on `connect` instead of in the constructor? If not, could you move both of them into the constructor so it's consistent? Feel free to convert the constructor into a single "options" parameter.

Reply:
I think I got confused when I was doing the original implementation for enableDebuggerAutoRecovery, and thought it had to be set in connect - just verified both flags work fine in the constructor.